src/test/scala/pl/allegro/tech/marathon/plugin/auth/BasicAuthCredentialsExtractorTest.scala

```diff
@@ -0,0 +1,21 @@
+package pl.allegro.tech.marathon.plugin.auth
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class BasicAuthCredentialsExtractorTest extends FlatSpec with Matchers {
+
+  val extractor = new BasicAuthCredentialsExtractor
+
+  "BasicAuthCredentialsExtractor" should "extract base64 credentials" in {
+    val actual = extractor.extractCredentials(
```

Comment:
Changed it, but I'm not sure if this is good idea for simple tests. It will be useful later. I hope this test is more scalish (scalic) now.